pages/index.vue

```diff
@@ -1,5 +1,19 @@
 <script setup lang="ts">
 import MainPageFeatures from "~/components/MainPage/MainPageFeatures.vue";
+defineOgImageComponent("RedotOG", {
+  title: "Redot Engine",
+  description: "Open‑source Multi-platform 2D and 3D game engine for everyone",
+  imageUrl: "",
+  backgroundColor: "#000000",
+  brandColor: "#FF6E01",
+});
+
```

Comment:
I think @reflexguru has a social banner image somewhere. Can that be used here for the home page?

Reply:
The color is not changing anything yet. And if we go for just a image, it might end up like:

```
defineOgImage({
  url: '/my-image.png'
})
```